SilMock/tests/Google/Service/Directory/Resource/MembersTest.php

```diff
@@ -133,16 +123,21 @@ public function testListMembersOnlyOwner()
                 ]
             );
         } catch (Exception $exception) {
-            self::fail(
-                sprintf(
-                    'Was expecting the members.list method to function, but got: %s',
-                    $exception->getMessage()
-                )
-            );
+            $this->failure($exception);
         }
         self::assertEmpty(
             $members->getMembers(),
             'Was expecting the members.list method to have no owner types.'
         );
     }
+
+    protected function failure(Exception $exception): void
+    {
+        self::fail(
+            sprintf(
+                'Was expecting the members.insert method to function, but got: %s',
```

Comment:
Looks like this might have been a typo, but I'm not sure: should `members.insert` here have been `members.list`? (as in the deleted lines)